lms/djangoapps/teams/static/teams/js/collections/team_membership.js

```diff
@@ -8,15 +8,24 @@
 
                     this.course_id = options.course_id;
                     this.username = options.username;
+                    this.privileged = options.privileged;
                     this.perPage = options.per_page || 10;
                     this.server_api['expand'] = 'team';
-                    this.server_api['course_id'] = function () { return encodeURIComponent(this.course_id); };
+                    this.server_api['course_id'] = function () { return encodeURIComponent(options.course_id); };
                     this.server_api['username'] = this.username;
                     delete this.server_api['sort_order']; // Sort order is not specified for the TeamMembership API
                     delete this.server_api['order_by']; // Order by is not specified for the TeamMembership API
                 },
 
-                model: TeamMembershipModel
+                model: TeamMembershipModel,
+
+                canUserCreateTeam: function() {
+                    // Note: non-privileged users are automatically added to any team
+                    // that they create. This means that if multiple team membership is
+                    // disabled that they cannot create a new team when they already
+                    // belong to one.
+                    return this.privileged || this.length === 0;
```

Comment:
We actually also want to allow staff (course staff and global staff) to create multiple teams. They also are not automatically added to the team.